apps/communications/contacts/js/views/settings.js

```diff
@@ -346,36 +346,56 @@ contacts.Settings = (function() {
   function doExport(strategy) {
     // Launch the selection mode in the list, and then invoke
     // the export with the selected strategy.
-    contacts.List.selectFromList(_('exportContactsAction'),
-      function onSelectedContacts(promise) {
-        // Resolve the promise, meanwhile show an overlay to
-        // warn the user of the ongoin operation, dismiss it
-        // once we have the result
-        requireOverlay(function _loaded() {
-          utils.overlay.show('preparing-contacts', null, 'spinner');
-          promise.onsuccess = function onSuccess(ids) {
-            // Once we start the export process we can exit from select mode
-            // This will have to evolve once export errors can be captured
-            contacts.List.exitSelectMode();
-            var exporter = new ContactsExporter(strategy);
-            exporter.init(ids, function onExporterReady() {
-              // Leave the contact exporter to deal with the overlay
-              exporter.start();
-            });
-          };
-          promise.onerror = function onError() {
-            contacts.List.exitSelectMode();
-            utils.overlay.hide();
-          };
-        });
-      },
-      null,
-      navigationHandler,
-      {
-        isDanger: false,
-        transitionLevel: EXPORT_TRANSITION_LEVEL
-      }
-    );
+
```

Comment:
A comment is needed here in order to document why we are trying here to get the number of FB Contacts